docs/src/_guide/mappers.md

```diff
@@ -0,0 +1,185 @@
+---
+title: Inline Data Mapping
+description: Alter records and streams on the fly.
+layout: doc
+weight: 5
+---
+
+Meltano gives you the flexibility to alter data passing through your EL pipelines to do precisely what you need for your use case.
+Although commonly users want to replicate their data in the most raw format, using [ELT vs ETL](https://meltano.com/blog/the-benefits-of-elt-vs-etl-what-you-need-to-know/
+), there are many use cases that require minor alterations to data on the fly.
+This is where mappers, or also referred to as inline stream maps, shine!
+
+# Features
+
+The mapper features can be viewed in 2 categories: stream level or property (record) level.
+Stream level features allow you to modify the stream itself, whereas the property level features allow you to modify contents of the stream.
+
+## Stream Level
+
+- Aliasing: streams can be aliased to provide custom naming downstream.
+- Filtering: streams records can be filtered based on any user-defined logic.
```

Comment:
```suggestion
- Filtering: stream records can be filtered based on any user-defined logic.
```

With the way we've got property level changes labeled as `property (record)` this sentence is suddenly very confusing to me. Would `property (column)` be more appropriate?

Reply:
@tayloramurphy I agree its confusing. I updated to column.